garage/sampler/__init__.py

```diff
@@ -1,8 +1,3 @@
-from garage.sampler.stateful_pool import SharedGlobal
 from garage.sampler.stateful_pool import singleton_pool
-from garage.sampler.parallel_sampler import truncate_paths  # noqa: I100
-from garage.sampler.stateful_pool import ProgBarCounter
 
-__all__ = [
-    "SharedGlobal", "singleton_pool", "truncate_paths", "ProgBarCounter"
```

Comment:
This module is in general a mess, so there are no great decisions here. This looks okay.

Why are the base classes not imported?

Reply:
done